File: python/vowels_constunant.py
str1 = input("Enter a string. ")
vowels=0
constunant=0

for i in str1:
    if(i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u' or i == 'A' or i == 'E' or i == 'I' or i == 'O' or i == 'U'):
        vowels += 1
    else:
        constunant += 1
print(f"There are {vowels} vowels and {constunant} constunants in the string")

File: flutterquestion/vowels_constunant.dart
import 'dart:io';

void main() {
  print("enter a string. ");
  String str1 = stdin.readLineSync();

  int vowels = 0;
  int constunants = 0;
  print(str1.length);
  for (int i = 0; i < str1.length; i++) {
    print(vowels);

    if (i == 'a' ||
        i == 'e' ||
        i == 'i' ||
        i == 'o' ||
        i == 'u' ||
        i == 'A' ||
        i == 'E' ||
        i == 'I' ||
        i == 'O' ||
        i == 'U') {
      vowels += 1;
    } else {
      constunants += 1;
    }
  }

  print("There are $vowels vowels and $constunants constunants in the string1");
}
